module/motion/KinematicsConfiguration/data/config/KinematicsConfiguration.yaml

```diff
@@ -27,7 +27,7 @@ head:
   # head movement limits
   limits: # pitch limit is a function of yaw ... need to calculate this properly
     yaw: [-pi/4, pi/4] # [min_yaw, max_yaw]
-    pitch: [pi/6, pi/6] # [min_pitch, max_pitch]
+    pitch: [0, pi/2] # [min_pitch, max_pitch]
```

Comment:
pi/2 is quite large, I think the robot would self collide before this. May be worth running a test where you either make the limit either the angle it collides with the torso, or where it can see it's torso

Reply:
Should also consider whether localisation balls or vision balls is more correct? Maybe both, however for different reasons/functionality